worker/upload.go

```diff
@@ -1178,114 +986,61 @@ func (s *slabUpload) launch(req *sectorUploadReq) (interrupt bool, err error) {
 		return false, nil
 	}
 
-	// find next candidate
-	var candidate *uploader
-	for _, uploader := range s.candidates {
-		if _, used := s.used[uploader.hk]; used {
+	// find candidate
+	var candidate *candidate
+	for _, c := range s.candidates {
+		if c.used {
 			continue
 		}
-		candidate = uploader
+		candidate = c
 		break
 	}
 
 	// no candidate found
 	if candidate == nil {
 		err = errNoCandidateUploader
-		interrupt = !req.overdrive && len(s.overdriving[req.sector.index]) == 0
+		interrupt = !req.overdrive && !s.ongoingOverdrive(req.sector.index)
```

Comment:
Slightly confused as to why this is called `interrupt` here and on the calling side `overdrive`.
Because it seems like the calling side interrupts when `!overdrive` which is kind of the opposite time `interrupt` is actually `true`.

Reply:
I forgot to update it on the caller side. It should be called `interrupt`, I wanted to get away from the negation on the caller's side. It was never a good name `overdrive` meant "there's an ongoing overdrive sector, so don't stop the upload just yet"